Tests/SentryTests/Helper/SentrySerializationTests.swift

```diff
@@ -0,0 +1,133 @@
+import XCTest
+
+class SentrySerializationTests: XCTestCase {
+
+    func testSentryEnvelopeSerializesWithSingleEvent() {
+        // Arrange
+        let event = Event()
+        
+        let item = SentryEnvelopeItem(event: event)
+        let envelope = SentryEnvelope(id: event.eventId, singleItem: item)
+        // Sanity check
+        XCTAssertEqual(event.eventId, envelope.header.eventId)
+        XCTAssertEqual(1, envelope.items.count)
+        XCTAssertEqual("event", envelope.items[0].header.type)
+        
+        // Act
+        let serializedEnvelope = serializeEnvelope(envelope: envelope)
+
+        // Assert
+        if let actual = SentrySerialization.envelope(with: serializedEnvelope) {
+            XCTAssertEqual(envelope.header.eventId, actual.header.eventId)
+            XCTAssertEqual(1, actual.items.count)
+            XCTAssertEqual("event", envelope.items[0].header.type)
+            XCTAssertEqual(envelope.items[0].header.length, actual.items[0].header.length)
+            XCTAssertEqual(envelope.items[0].data, actual.items[0].data)
+        } else {
+            XCTFail("Could not deserialize envelope.")
+        }
+    }
+
+    func testSentryEnvelopeSerializesWithManyItems() {
+        // Arrange
+        let itemsCount = 15
+        var items: [SentryEnvelopeItem] = []
+        for i in 0..<itemsCount {
+            let bodyChar = "\(i)"
+            let bodyString = bodyChar.padding(
+                toLength: i + 1,
+                withPad: bodyChar,
+                startingAt: 0)
+
+            let itemData = bodyString.data(using: .utf8) ?? Data()
```

Comment:
L: is it possible to void `??` and do like in Kotlin if something can be null `!!`? if something should not be null and it is, it should fast fail

Reply:
Yes, it is possible. The result of `bodyString.data(using: .utf8)` should never be `nil`. I'm going to change it.

Reply:
I added a separate `.swiftlint` config file just for the tests, because we don't allow `force_unwrapping` in Swift code, which makes sense IMO, but not for the tests.